src/scripts/main.js

```diff
@@ -1,8 +1,28 @@
 'use strict';
 
 const people = require('./lib/people');
-
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashboard = document.getElementsByClassName('dashboard');
+const currentTable = dashboard[0].getElementsByTagName('tbody')[0];
```

Comment:
You can use querySelector() to find [0] elements

Reply:
yes, you  are right! thank you